### PR TITLE
updates:dashboard

### PR DESCRIPTION
**Pull request by matthewcarleton**
just a simple one to test :)

### CODE REVIEW

File: sass/components/ClusterOverview/health.scss [hunk 1]
```diff
@@ -1,16 +1,30 @@
-.kubevirt-health__body {
-  text-align: center;
-  max-height: 130px;
+/* stylelint-disable plugin/selector-bem-pattern */
+.kubevirt-cluster-overview__health-grid {
```
Comment (suomiy) [inline sass/components/ClusterOverview/health.scss]: can we update `StorageOverview/ocs-health.scss` for consitency? or unite these two together into one file?

Comment (rawagner) [reply sass/components/ClusterOverview/health.scss]: @matthewcarleton I've updated snapshots etc to make the build pass. As @suomiy said, can you please update `ocs-health.scss` in this PR too ? For other changes I'd suggest to create a new branch and new PR to divide these changes to smaller pieces.

Comment (matthewcarleton) [reply sass/components/ClusterOverview/health.scss]: Yes, I can do that. Thanks @suomiy

Comment (matthewcarleton) [reply sass/components/ClusterOverview/health.scss]: Sorry this got larger than it should have. I'll create a new PR for the following updates.

**Pull request merged**